wiki/People/Staff_Log/en.md

```diff
@@ -21,659 +21,659 @@ Using the articles listed below, you can find all Beatmap Nominators and their s
 
 ### January
 
--   Added [\[Shi-Ra\]](/users/%5BShi-Ra%5D) as osu!mania BAT (2014-01-01)
--   Added [TKS](/users/TKS) as Taiko BAT (2014-01-04)
--   Readded [Teara](/users/Teara) as BAT (2014-01-05)
--   Added [shARPII](/users/shARPII) as Chat Moderator (2014-01-16)
--   Added [Stickie](/users/Stickie) as Chat Moderator (2014-01-16)
--   Removed [Gabe](/users/Gabe) as BAT (2014-01-17)
--   Added [Krah](/users/Krah) as CtB BAT (2014-01-20)
--   Readded [dkun](/users/dkun) as BAT (2014-01-20)
--   Moved [Breeze](/users/Breeze) to Alumni (2014-01-21)
--   Removed [HeatKai](/users/HeatKai) as BAT (2014-01-21)
--   Moved [jericho2442](/users/jericho2442) to Alumni (2014-01-21)
--   Moved [lolcubes](/users/lolcubes) to Alumni (2014-01-21)
--   Moved [MMzz](/users/MMzz) to Alumni (2014-01-21)
--   Moved [Sync](/users/Sync) to Alumni (2014-01-21)
--   Moved [those](/users/those) to Alumni (2014-01-21)
--   Moved [YGOKid8](/users/YGOKid8) to Alumni (2014-01-21)
--   Readded [YGOKid8](/users/YGOKid8) as BAT (2014-01-23)
--   Moved [Sakura](/users/Sakura) to Alumni (2014-01-24)
--   Moved [Kurai](/users/Kurai) to GMT (2014-01-25)
--   Added [HelloSCV](/users/HelloSCV) as BAT (2014-01-27)
+-   Added [\[Shi-Ra\]](/users/2008489) as osu!mania BAT (2014-01-01)
+-   Added [TKS](/users/940878) as Taiko BAT (2014-01-04)
+-   Readded [Teara](/users/123491) as BAT (2014-01-05)
+-   Added [shARPII](/users/776257) as Chat Moderator (2014-01-16)
+-   Added Stickie as Chat Moderator (2014-01-16)
+-   Removed [Gabe](/users/654108) as BAT (2014-01-17)
+-   Added [Krah](/users/1436748) as CtB BAT (2014-01-20)
+-   Readded [dkun](/users/154400) as BAT (2014-01-20)
+-   Moved [Breeze](/users/77537) to Alumni (2014-01-21)
+-   Removed [HeatKai](/users/332555) as BAT (2014-01-21)
+-   Moved [jericho2442](/users/88904) to Alumni (2014-01-21)
+-   Moved [lolcubes](/users/578401) to Alumni (2014-01-21)
+-   Moved MMzz to Alumni (2014-01-21)
+-   Moved [Sync](/users/806921) to Alumni (2014-01-21)
+-   Moved those to Alumni (2014-01-21)
+-   Moved [YGOKid8](/users/users/69114) to Alumni (2014-01-21)
```

Comment:
`/users/users` ?

Reply:
:eyes: fixed for the two other instances